src/main/java/br/com/budgetboys/pf_finance_service/adapters/inbound/controller/income/IncomeController.java

```diff
@@ -0,0 +1,62 @@
+package br.com.budgetboys.pf_finance_service.adapters.inbound.controller.income;
+
+import br.com.budgetboys.pf_finance_service.adapters.outbound.service.income.IncomeService;
+import br.com.budgetboys.pf_finance_service.domain.income.IncomeCreateDTO;
+import br.com.budgetboys.pf_finance_service.domain.income.IncomeResponseDTO;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("income")
+public class IncomeController {
+
+    private final IncomeService incomeService;
+
+    public IncomeController(IncomeService incomeService) {
+        this.incomeService = incomeService;
+    }
+
+    @PostMapping("create")
+    public ResponseEntity<?> create(@RequestBody IncomeCreateDTO income) {
+        try {
+            IncomeResponseDTO savedIncome = this.incomeService.saveIncome(income);
+            return ResponseEntity.ok(savedIncome);
+        } catch (IllegalArgumentException e) {
+            return ResponseEntity.badRequest().body(e.getMessage());
+        } catch (Exception e) {
+            return ResponseEntity.internalServerError().body(e.getMessage());
+        }
+    }
+
+    @GetMapping("find")
```

Comment:
Transformar rota para o padrão RESTful, para ser utilizado desta forma: `api/incomes/1`
```suggestion
    @GetMapping("/{id}")
```